packages/core/admin/admin/src/pages/SettingsPage/pages/TransferTokens/EditView/components/FormTransferTokenContainer/index.js

```diff
@@ -16,6 +17,30 @@ const FormTransferTokenContainer = ({
 }) => {
   const { formatMessage } = useIntl();
 
+  const typeOptions = [
+    {
+      value: 'push',
+      label: {
+        id: 'Settings.transferTokens.types.push',
+        defaultMessage: 'Push',
+      },
+    },
+    {
+      value: 'pull',
+      label: {
+        id: 'Settings.transferTokens.types.pull',
+        defaultMessage: 'Pull',
+      },
+    },
+    {
+      value: 'push-pull',
+      label: {
+        id: 'Settings.transferTokens.types.pull',
+        defaultMessage: 'Push & pull',
```

Comment:
@marcoautiero any suggestion on the text to show in the token type selection when you want a token to transfer with push and pull?

Reply:
"Push and pull"